intel_extension_for_transformers/llm/inference/inference.py

```diff
@@ -228,9 +230,16 @@ def __call__(
         return False
 
 
-def max_input_len(model, outlen=0):
-    # need to adjust due to perf and real usage
-    return 128
+def max_input_len(input_text_length):
```

Comment:
was this change introduced by Yi?

Reply:
This is not our modification, it is the latest update in branch main.